specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/QueryPackQueries_API.json

```diff
@@ -0,0 +1,464 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/QueryPackNameParameter"
+          },
+          {
+            "$ref": "#/parameters/QueryIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "The query has been successfully removed from the Log Analytics QueryPack"
+          },
+          "204": {
+            "description": "The specified query or associated QueryPack does not exist."
+          }
+        },
+        "x-ms-examples": {
+          "QueryDelete": {
+            "$ref": "./examples/QueryPackQueriesDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
+      "description": "Describe the format of an Error response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "QueriesResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "properties": {
+          "$ref": "#/definitions/LogAnalyticsQueryPackQueryProperties"
+        }
+      },
+      "description": "An Azure resource QueryPack-Query object"
+    },
+    "LogAnalyticsQueryPackQuery": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties that define an Log Analytics QueryPack-Query resource.",
+          "$ref": "#/definitions/LogAnalyticsQueryPackQueryProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/QueriesResource"
+        }
+      ],
+      "description": "A Log Analytics QueryPack-Query definition."
+    },
+    "LogAnalyticsQueryPackQueryProperties": {
+      "description": "Properties that define an Log Analytics QueryPack-Query resource.",
+      "properties": {
+        "queryId": {
+          "type": "string",
+          "description": "The unique ID of your application. This field cannot be changed.",
+          "readOnly": true
+        },
+        "displayName": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Unique name for your Query Pack."
```

Comment:
>Unique name [](start = 26, length = 11)

is this truly unique?  If so, at what scope?

Reply:
the scope is the query pack.